src/com/opc/opcconnectionimpl.cpp

```diff
@@ -39,9 +43,18 @@ void COpcConnectionImpl::disconnect(){//const char* paGroupName){
   DEVLOG_INFO("COpcConnectionImpl disconnect\n");
   if (isConnected())
   {
+      if (mOpcHost) {
```

Comment:
The indention seems to be off (4 spaces, but should only be 2 spaces)
I would prefer `mOpcHost != nullptr` as condition, as I do not need to think how the boolean value is generated for the condition